readthedocs/projects/models.py

```diff
@@ -352,9 +352,20 @@ class Project(models.Model):
         default='',
         blank=True,
         help_text=_(
-            'Path from project root to <code>conf.py</code> file '
-            '(ex. <code>docs/conf.py</code>). '
-            'Leave blank if you want us to find it for you.',
+            "Path from project root to <code>conf.py</code> file "
+            "(ex. <code>docs/conf.py</code>). "
+            "Leave blank if you want us to find it for you.",
+        ),
+    )
+    rtd_conf_file = models.CharField(
+        _(".readthedocs.yml configuration file"),
+        max_length=255,
+        default="",
+        blank=True,
```

Comment:
I think we need this to be nullable, at least for the initial deploy, since we run multiple webs and it will break existing project editing in the timeframe between migration & full code deploy.

I think in general also `None` is a more explicit default here than `""`, so I think I'm fine keeping it nullable with a default `None`?

Reply:
Agreed! Implementing!